packages/commonwealth/client/scripts/state/api/members/removeRole.ts

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+import { notifyError } from 'controllers/app/notifications';
+import app from 'state';
+import RoleInfo from '../../../models/RoleInfo';
+
+type RemoveRoleProps = {
+  role: RoleInfo;
+  onRoleUpdate: (oldRole: RoleInfo, newRole: RoleInfo) => void;
+};
+
+const removeRole = async ({ role, onRoleUpdate }: RemoveRoleProps) => {
```

Comment:
remove role and upgrade role should be react-query mutations. Take a look on other files. Structure of the files should always look the same, so:
1. types/interfaces
2. function
3. hook